src/command/identity/import.ts

```diff
@@ -0,0 +1,104 @@
+import Wallet from 'ethereumjs-wallet'
+import { readFileSync, statSync } from 'fs'
+import { Argument, LeafCommand, Option } from 'furious-commander'
+import { exit } from 'process'
+import { IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+
+export class Import extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'import'
+
+  public readonly description = 'Import V3 wallet as a new identity'
+
+  @Argument({ key: 'path', required: true, describe: 'Path to the V3 wallet file' })
+  public path!: string
+
+  @Option({ key: 'identityName', describe: 'Name of the identity to be saved as' })
+  public identityName!: string
+
+  @Option({ key: 'password', describe: 'Password for the V3 wallet' })
+  public password!: string
+
+  public async run(): Promise<void> {
+    this.initCommand()
+    this.checkForValidPath()
+    await this.ensurePasswordIsProvided()
+    const data = readFileSync(this.path).toString()
+    const wallet: Wallet = await this.decryptV3Wallet(data)
+    await this.ensureIdentityNameIsProvided()
+    await this.saveWallet(wallet)
+    this.console.log(`V3 Wallet imported as identity '${this.identityName}' successfully`)
+  }
+
+  /** Init additional properties of class, that are not handled by the CLI framework */
+  private initCommand(): void {
+    super.init()
+  }
+
+  private async decryptV3Wallet(data: string): Promise<Wallet> {
+    try {
+      const wallet: Wallet = await Wallet.fromV3(data, this.password)
+
+      return wallet
+    } catch (error) {
+      this.console.error('Failed to decrypt wallet:\n' + error.message)
+
+      exit(1)
+    }
+  }
+
+  private async ensurePasswordIsProvided(): Promise<void> {
+    if (!this.password) {
+      this.console.log('You have not defined the password with the "--password" option.')
+      this.password = await this.console.askForPassword('Please provide the password for this V3 Wallet')
+    }
+  }
```

Comment:
I think this function will be needed outside of this class as well when we use identities for example for `feed` uploads.

_See comment: root-command-inheritance_